frontend/src/scenes/session-recordings/settings/SessionRecordingSettings.tsx

```diff
@@ -69,6 +71,28 @@ export function SessionRecordingSettings({ inModal = false }: SessionRecordingSe
                 </p>
             </div>
 
+            {featureFlags['hackathon-apm'] && (
+                <div className="space-y-2">
+                    <LemonSwitch
+                        data-attr="opt-in-capture-performance-switch"
+                        onChange={(checked) => {
+                            updateCurrentTeam({ capture_performance_opt_in: checked })
+                        }}
+                        label="Capture network requests and performance information within recordings"
+                        labelClassName={inModal ? 'text-base font-semibold' : ''}
+                        bordered={!inModal}
+                        fullWidth={inModal}
+                        checked={!!currentTeam?.capture_performance_opt_in}
+                        disabled={!currentTeam?.session_recording_opt_in}
```

Comment:
I think we could just remove this disabled line as the performance capture is not tightly coupled in the JS